### PR TITLE
Proposal for event and eventName

### LINKED ISSUE #2382: Consideration of eventName element?
Respectfully, I would like to ask the Council to consider the value of creating the eventName element. Increasingly, the work I'm doing on historical performance and business events (sorry ...) calls out for an eventName tag that would provide parity with named people, places, organization as connected together in a time or time span through a named occurrence. I hope I'm not the only one (and suspect I'm not).

In 2021 there was a back-and-forth on the listserv about its value: 

- https://listserv.brown.edu/cgi-bin/wa?A2=ind2103&L=TEI-L&P=R8274
- https://listserv.brown.edu/cgi-bin/wa?A2=ind1706&L=TEI-L&P=R20066

Note that in one of Torsten Schassan's contributions to that thread he calls upon us "for the sake of completeness and in analogy to other cases, there might be the need to introduce another element to the TEI, <eventName>." and then goes on to provide considerations.

Here is a (particularly gnarly and complex) example (but indicative of the kind of work that I think would benefit from the addition of eventName):
@JanelleJenstad may be happy to see this excerpt from Stow - it's from REED London and sitting in LEAF-Writer, so I've stripped out the @refs for readability sake (and with apologies in advance for tag abuse). In this particular case there are three named events within the text: the marriage, the feast, and the masque. 

`<ab>The <date when="1613-12-26" cert="high">26. of December, 1613</date> <persName key="Carr, Robert" cert="high" type="real">Robert earle of Somerset</persName> married the lady
            <persName key="Howard, Frances" cert="high" type="real">Francis Howard</persName> daughter to ye <persName key="Howard, Thomas" cert="high" type="real">earle of Suffolke</persName> <gap reason="omitted"/> &amp; vpon <date when="1614-01-04" cert="high">Tewsday the 4. of
               Ianuary</date>, the Bride, &amp; Bridegroome, being accompanied w<ex>i</ex>t<ex>h</ex> the <persName key="Stuart, Esmé" cert="high" type="real">duke of Lenox</persName>,
            the <persName key="Howard, Henry" cert="high" type="real">Lord priuie Seale</persName>, the <persName key="Howard, Thomas" cert="high" type="real">lord Chamberlayne</persName>, the <persName key="Somerset, Edward" cert="high" type="real">earles of Worcester</persName>, <persName key="Herbert, William" cert="high" type="real">Pembroke</persName>,
            <persName key="Herbert, Philip" cert="high" type="real">Montgomery</persName>, &amp; others, and with many honorable Barons, knights, &amp; gentlemen
            of qualitie, came to <placeName key="Merchant Taylors’ Hall" role="placeEntertainment" type="hall" cert="high">marchant-taylers hall</placeName>, where the Lord Maior, &amp;
            Aldermenne, of London, in their Scarlet robes, entertayned them with hearty
            welcome, &amp; feasted them with all magnificence: <gap reason="omitted"/> &amp; about 3. a clock in
            the morning they returned to <placeName key="Whitehall Palace" role="placeEntertainment" type="palace" cert="high">white hall,</placeName> and vppon twelfe night there was a maske
            of gentlemen of Grayes Inne<gap reason="omitted"/>.</ab>
`

Thank you for your consideration!

### PR DESCRIPTION
**Pull request by skurzinz**
Fixes #2382.

Following the checklist @sydb prepared in https://github.com/TEIC/TEI/issues/2382#issuecomment-1475392687, and going a bit into detail on the whole `<event>` discussion beyond  `<eventName>`, we proceeded in our attempt to not only include this new element but also align the `<event>` element with other nameable elements that are used to create *ographies*. 

- [x] Figuring out where in chapter 13 we would introduce & discuss this new element, 
- [x] and write the new prose, including an example.

In our PR it is located between `<placeName>` and `<objectName>` in the Names section. 

- [x] Figure out what other parts of ND need to be re-written,
- [x] and alter them.

We tried to be minimally invasive, but moved the description of `<event>` in general out of the `<person>` section while keeping the useful information on *Personal Events* in place. This results in a new section on events on the same level as `<person>`, `<org>`, `<place>`, **`<event>`** and `<object>`. 

- [x] Write new tagdoc for `<eventName>`, including an example,
- [x] and remember to include it (via `<xi:include>`) in ND.

An attempt at the latter is contained in https://github.com/TEIC/TEI/commit/62b1b9a35ac9227c407653246c6a0131f5c499ad, but is this enough? 

- [x] Change content model of `<event>` so that `<eventName>` can appear.
- [x] Find places that exemplify other encoding that would now use `<eventName>`, including examples of event/head and event/label, and consider changing them.

With regard to this we did not alter the existing examples, which use `<head>` and `<label>` interchangably, yet provided example use cases for (“canonically“) named events that may even have authority file data associated. This keeps it open what editors choose to label events. 

In addition, we changed the content model of `<listEvent>` and `<event>` to allow other lists to be contained in them. The rationale was to align the `<event>` with other *ography* elements: A `<listPerson>` contained in an `<event>` may be used to list persons who were involved in it (e.g. A, B and C were taking part in the nth session of parliament), a `<listPlace>` may be used to describe locations of an event (e.g. the filming of *The Third Man* was done in Vienna and London), and similar relations.

We are first creating this PR and will shortly follow up on this on TEI-L.

### CODE REVIEW

**Comment by skurzinz** [comment]
For reference, @helmutwklug and @onbcst were conceptionally involved in the discussion leading to the proposed changes.

**Comment by onbcst** [comment]
Thanks for drafting the pull request. From my point of view allowing `<listPerson>` and `<listPlace>` in `<event>` makes totally sense but I think they are not necessary as child elements of `<listEvent>`. Therefore I suggest removing 
```xml
<alternate minOccurs="0" maxOccurs="unbounded">
  <classRef key="model.personLike" minOccurs="1" maxOccurs="unbounded"/>
  <elementRef key="listPerson" minOccurs="1" maxOccurs="unbounded"/>
</alternate>
<alternate minOccurs="0" maxOccurs="unbounded">
  <classRef key="model.placeLike" minOccurs="1" maxOccurs="unbounded"/>
  <elementRef key="listPlace" minOccurs="1" maxOccurs="unbounded"/>
</alternate>
```
again from listEvent.xml. It would also match the behavior similar to `listPerson` and `listPlace` which also allow only nested lists from the same type.

**Comment by skurzinz** [comment]
Thanks @onbcst, fully agreed. That was a lapsus on my side.

**Comment by sydb** [comment]
Wow, nice job!

(I have to admit, I did not actually _expect_ someone to follow the checklist, but am very pleased you did!)

I have not had a chance to look at this carefully,[2] but can quickly sketch out some next steps. I have followed each step with my suggestion for who should do that work, but that is just my off-the-cuff idea, not anything carved in stone. Details on some of the tasks are listed below the list of them.

1. Fix double inclusion of the `<elementSpec>`s for  `<event>`, `<eventName>`, and `<listEvent>` (proposers)
2. Fix duplicate IDs in Guidelines encoding (proposers)
3. Figure out when `<eventName>` ends up in model.nameLike twice, and fix (Council)
4. If `<listEvent>` and `<event>` are still undefined, figure out why and fix (Council)
5. Fix `<listEvent>` content model (it’s ambiguous) (Council)
6. Generate readable _Guidelines_ and schemas (Council)
7. Read them and hammer out details (both)

**1. Fix double inclusion of the `<elementSpec>`s for  `<event>`, `<eventName>`, and `<listEvent>` (proposers)**

@skurzinz correctly worried if commit 62b1b9a was correct, but was worried it was not enough — the problem is it is too much. 😄  The `<eventName>` element is included twice, both from ND, once from line 771, and then again from line 2344. Same is true for `<evnt>` and `<listEvent>`, which are included from both #DCCAHPA and #DNDEVNT.

**2. Fix duplicate IDs in Guidelines encoding**
The first round of validity errors are all duplicate values of `@xml:id`:
~~~xml
    <ERROR>id 'SecondDefPrague' used more than once</ERROR>
    <ERROR>id 'Prague' used more than once</ERROR>
    <ERROR>id 'ThirtyYearsWar' used more than once</ERROR>
    <ERROR>id 'BattleofRocroi' used more than once</ERROR>
    <ERROR>id 'Rocroi' used more than once</ERROR>  [1]
    <ERROR>id 'event01' used more than once</ERROR> [1]
    <ERROR>id 'event02' used more than once</ERROR> [1]
    <ERROR>id 'event03' used more than once</ERROR> [1]
~~~
[1] These errors went away when I removed all of DNDEVNT.

**3. Figure out when `<eventName>` ends up in model.nameLike twice, and fix**
**4. If `<listEvent>` and `<event>` are still undefined, figure out why and fix**

These two will likely go away when (1) is fixed, but someone should double-check.

**5. Fix `<listEvent>` content model**

Because `<event>` is a member of model.eventLike, the two cannot occur next to each other in content (unless both are limited to a specific number of occurences).

**6. Generate readable _Guidelines_ and schemas**

Either with or without CI server.

**7. Read them and hammer out details**

Probably the hardest part. 😄 

[2]  Because when I first saw this I had < 18 hours before leaving for the TEI Council meeting. I am currently in the airport. ✈️

**Comment by skurzinz** [comment]
Many thanks @sydb for your comments and further tasks. 

1. is done in [14376d5](https://github.com/TEIC/TEI/pull/2427/commits/14376d562ac6fdb9da84e1a5528bb282012fa9ea). I am marking this as checked in the checklist above.

2. above is that I used the same example both for the `<eventName>` and `<event>` specs, just copypasting.  
This being the first time I dig into the Guidelines, I did not find a quick way of instead referencing it in one of the places. May I ask any of you to just fix that or tell me how to best address this?

**Comment by sydb** [comment]
[Wow. You’re fast. I am going to have trouble keeping up.]
 
**1.**
Excellent.

**2.**
First, no criticism (at least not from me). For your first dive into editing the _Guidelines_, you are doing a marvelous job. I am impressed.
Second, I am sorry to say I don’t think there _is_ any way to reference an example (so that the same example is encoded once but appears in 2 places). The best practice, of course, is to come up with a different example. But often you will find that the same example is copied _except_ for the ID, which is slightly different. Even with a different (but similar) example, one often wishes to use the same ID, and we just can’t. For example, there are three examples of `<object>` with the ID values "Alfred-Jewel", "AlfredJewel", and "Alfred_Jewel".
Third, I would be happy to do that, but I would not write-access to your repo, no?

**Comment by skurzinz** [comment]
2. is now fixed in [7d8a834](https://github.com/TEIC/TEI/pull/2427/commits/7d8a834f9657327572a89f4a1ebddcc718f0a3db) by swapping out the Thirty Years’ War example against a World War II example with multiple `@xml:lang` variants, thus hinting at another possible set of event naming issues for encoders to consider.

**Comment by skurzinz** [comment]
> Third, I would be happy to do that, but I would not write-access to your repo, no?

I allowed maintainer changes on merging the PR, but didn’t use that myself. Just to be sure I invited @sydb as a collaborator in my fork. 

From the task list above, I don’t see any further things for us as proposers to work on currently – much looking forward to the *Read them and hammer out details* step, while continuing to monitor this…

File: P5/Source/Specs/event.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="event">
   <gloss xml:lang="en" versionDate="2008-12-09">event</gloss>
   <gloss versionDate="2008-12-09" xml:lang="fr">évènement</gloss>
-  <desc versionDate="2007-08-01" xml:lang="en">contains data relating to any kind of significant event associated with a person, place, or organization.</desc>
+  <desc versionDate="2007-08-01" xml:lang="en">contains data relating to any kind of significant event associated with a person, place, organization, or other event.</desc>
```
Comment (ebeshero) [inline P5/Source/Specs/event.xml]: Remember to change the `@versionDate`

File: P5/Source/Specs/eventName.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+$Id$
+--><?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="eventName">
+  <gloss versionDate="2023-05-02" xml:lang="en">name of an event</gloss>
+  <gloss versionDate="2023-05-02" xml:lang="de">Name eines Ereignisses</gloss>
+  <desc versionDate="2023-05-02" xml:lang="en">contains a proper noun or noun phrase used to refer to an event.</desc>
+  <desc versionDate="2023-05-02" xml:lang="de">enthält einen Eigennamen in Form eines Nomens oder einer Nominalphrase, der verwendet wird, um auf ein Ereignis zu verweisen.</desc>
+  <classes>
+    <memberOf key="att.datable"/>
+    <memberOf key="att.editLike"/>
```
Comment (ebeshero) [inline P5/Source/Specs/eventName.xml:16]: Add  `<memberOf key="att.canonical"/>` so it gets `@ref` please.

**Review by ebeshero — COMMENTED** [review]
@skurzinz Council F2F at Guelph is very, very impressed with this, and we think it won't take much work to fix the small problems noted here and include in the next release. We want to try to create some more examples, too in process.

File: P5/Source/Specs/event.xml (hunk 1 above)
Comment (skurzinz) [reply P5/Source/Specs/event.xml]: done in 8ddc8092

File: P5/Source/Specs/eventName.xml (hunk 2 above)
Comment (skurzinz) [reply P5/Source/Specs/eventName.xml:16]: done in 24f6e72b (using `att.personal` which includes `att.canonical` and `att.naming` (`@role` might be useful, and frankly I checked in `objectName.xml` how @jamescummings had done that for `object`s – which was the template for `<eventName>`).

**Comment by skurzinz** [comment]
[ea40022](https://github.com/TEIC/TEI/pull/2427/commits/ea400227690d7a2fe4bedb7676ff1dde58c9eb4e) adds `<object>`s as possible agents within an `<event>`, which I had overlooked. 
Example: The `<objectName ref="#MinsterLovellJewel">Minster Lovell Jewel</objectName>` from `objectName.xml` [here](https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-objectName.html#index-egXML-d54e111789) is discovered, and this may be formalized as an `<event>` including both the `<person>` in the active role of discovery and the `<object>` in the passive role.

**Comment by skurzinz** [comment]
> Council F2F at Guelph is very, very impressed with this, and we think it won't take much work to fix the small problems noted here and include in the next release. 

Good to hear, but: The Council should be satisfied with its working community, the annual conference and itself! The conversation we had during and after the panel at TEI2019 in Graz already included most of the points of this PR and set the agenda. During the panel discussion, AFAIR @martinascholger, @lb42, @martindholmes, @jamescummings, @sydb (to name who I remember from an event from 4 years back) were positive about the general idea and raised a few details that would have to be worked out. It only took us some time to sit down and do that.

Numerous discussions with @dasch124, @sennierer and @csae8092 and others contributed as well—credit where it’s due.

> We want to try to create some more examples, too in process.

If real life examples are needed I can also provide snippets from the @oeaw-ministerratsprotokolle for named governmental meeting session events (currently using `<label>` in lieu of `<eventName>`). Any suggestions on what kind of other examples may be useful?

**Comment by sydb** [comment]
> Any suggestions on what kind of other examples may be useful?

It would seem to me a crime not to use the TEI Conference in Graz as an example. 😃

**Comment by sydb** [comment]
I have just pushed 6c68379a7 to your repo, @skurzinz, without having accepted your invitation. I did not know that was possible; thank you.

In any case, this commit fixes the ambiguities from the `<event>` and `<listEvent>` content models while retaining the same set of constraints (I hope). It passes all the build tests I can run on my little laptop here. Presuming it passes the rest of the tests (which I can run when I get home), I think we are done with the list of fixes. 

In which case, the only thing left to do before generating outputs (which I forgot to mention previously) is to merge TEIC/TEI:dev into skurzinz:dev, so the latest changes are there.

**Comment by skurzinz** [comment]
> In any case, this commit fixes the ambiguities from the `<event>` and `<listEvent>` content models while retaining the same set of constraints (I hope).

I hope so too, the idea being that `<event>` should have 0-n possible `child::*[name()=('person','event','place','object','org')]` and their respective `list*` forms. As I said I was just copying over from the `object` models and did some possibly poisonous changes :)

**Comment by skurzinz** [comment]
> In which case, the only thing left to do before generating outputs (which I forgot to mention previously) is to merge TEIC/TEI:dev into skurzinz:dev, so the latest changes are there.

Thanks @sydb for reminding about that, of course that’s necessary. Seemingly, all the maintainers of the upstream repo are able to push to the PR, but in any case: Let’s not forget that. Many thanks!

**Comment by skurzinz** [comment]
> It would seem to me a crime not to use the TEI Conference in Graz as an example.

Done in [3e69268](https://github.com/TEIC/TEI/pull/2427/commits/3e692685c2fc285347217f44358ebb30ece015a2), could be replicated in `event.xml` as well.

**Comment by sydb** [comment]
Latest commit tweaks the content models and examples so that almost all the examples are now valid. The sole exception is “Platter”, Filip Fabricius’ additional name in the tagdoc for the `<event>` element, which is still invalid. (The `<addName>` is not valid as a direct child of `<person>`; it would be valid as a direct child of `<persName>` as in
~~~xml
        <persName>
          <forename>Filip</forename>
          <surname>Fabricius</surname>
          <addName>Platter</addName>
        </persName>
~~~
but if we did that, then the encoding of Filip’s name would not be parallel to that of Jaroslav’s and Vilém’s, and I did not know if you would want to change those, too, or nuke the “Platter”.)

And indeed I do have write access to your repo (as demonstrated by these commits), and I do not mind doing the work of merging in the latest from TEIC/TEI:dev myself. However, I do not actually know _how_ to do that.

As soon as “Platter” is fixed and TEIC/TEI:dev is merged in (whether someone else does it or someone tells me how to, which I would mildly prefer, because I really should know this, eh?) I can create outputs for us to play with.

**Comment by skurzinz** [comment]
With 4d39c1c8 above my branch is in sync with upstream [TEIC:dev](https://github.com/TEIC/TEI).

I already took care of merging those upstream changes back, but for reference or if more commits get added there in the meantime: I got these by merging TEIC:dev using a new PR https://github.com/skurzinz/TEI/pull/1 (just do a new PR). 

Alternatively, this could be done locally along the lines of https://stackoverflow.com/questions/12921125/git-merge-branch-of-another-remote (add another remote, pull or merge into a new local branch, merge new local branch into the fork:dev, commit this).

**Comment by sydb** [comment]
See the generated [_Guidelines_](https://bauman.zapto.org/~syd/temp/tei/Guidelines-web/en/html/) and [Exemplars](https://bauman.zapto.org/~syd/temp/tei/Exemplars/). (Note: that is a temporary location, and the outputs there will likely be replaced by new versions as we move this issue along, and likely will be deleted once this PR is merged.)

**Comment by skurzinz** [comment]
<del>Remark (which may be a new issue) while checking this (thanks to @sydb for preparing the previews!): </del>

<del>In current P5, `<note>` seems allowed as a child of `<org>` but not the other entity-like elements (person, place, event, object, relation, [name]). Is that intentional? What’s the reasoning behind it? </del>

Edit: sorry, I was lost in my tests…

**Comment by ebeshero** [comment]
@skurzinz Indeed, `<note>` is permitted as a child of the other "ography" elements (`<person>`, `<place>`, etc) in the NamesDates and Core modules. https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-note.html I rely on this extensively in the "listography" files for my projects. 

So we have no new issue here!

**Comment by skurzinz** [comment]
> So we have no new issue here!

@ebeshero you are right, it was a positional thing: I tested with a `<note>` after several `<list*>` in an `<event>` and that did not validate. Withdrawing my comment from above…

**Comment by skurzinz** [comment]
e2c432f (referring to my comment on treaties as events at https://github.com/TEIC/TEI/issues/2382#issuecomment-1525143951) and 1f1aaa4 are addressing minor issues that came to mind when deleting my own comments in d0fa3c2, hopefully improving guidance and possible ways of encoding `<event>`s. 

Other than that, I tested some examples I could come up with using the tei_all.rng created by @sydb and did not find any issues, but I may be blind by being directly involved, so I am pinging @onbcst @helmutwklug @ChristianeFritze once more to check their examples (I just made the stuff in our `Events_Beispieldaten` in the onb repo valid, but you may have more) against the temporary https://bauman.zapto.org/~syd/temp/tei/Exemplars/tei_all.rng and report back any issues.

My next commit will include a real life example drawn from the @oeaw-ministerratsprotokolle added to `Specs/event.xml` as promised in https://github.com/TEIC/TEI/pull/2427#issuecomment-1537543038.

**Comment by sydb** [comment]
I pulled from the TEIC repo (turned out to be nothing needed), have tweaked things a bit, and regenerated both [Guidelines](https://bauman.zapto.org/~syd/temp/tei/Guidelines-web/en/html/) and [Exemplars](https://bauman.zapto.org/~syd/temp/tei/Exemplars/).

Note that in e80feacde I also removed the extraneous new "es" `<desc>`. (As the content of the two was precisely the same, I presumed the error was adding the new one, as opposed to failing to delete the old one. If I am wrong about that, a Spanish speaker actually checked that description against the English version on 2023-05-07, whoever did that should just update the date.)

Is it maybe time to get multiple eyes reading this and (as I put it earlier) “hammer out the details”?

**Comment by sydb** [comment]
Noting that some (potential) changes have been made, I have just regenerated both [Guidelines](https://bauman.zapto.org/~syd/temp/tei/Guidelines-web/en/html/) and [Exemplars](https://bauman.zapto.org/~syd/temp/tei/Exemplars/).

File: P5/Source/Specs/event.xml [hunk 3]
```diff
@@ -8,18 +8,17 @@
...
+  <gloss versionDate="2008-12-09" xml:lang="en">event</gloss>
+  <gloss versionDate="2023-05-07" xml:lang="de">Ereignis</gloss>
   <gloss versionDate="2008-12-09" xml:lang="fr">évènement</gloss>
-  <desc versionDate="2007-08-01" xml:lang="en">contains data relating to any kind of significant event associated with a person, place, or organization.</desc>
-  <desc versionDate="2007-12-20" xml:lang="ko">개인, 장소, 또는 조직과 관련된 중요한 사건에 해당하는 데이터를 포함한다.</desc>
+  <desc versionDate="2023-05-07" xml:lang="en">contains data relating to any kind of significant event associated with a person, organization, place, object, or other event.</desc>
   <desc versionDate="2007-05-02" xml:lang="zh-TW">任何未必是口說的或溝通上的現象或發生情況，例如偶然的噪音或其他影響溝通的事件。</desc>
+  <desc versionDate="2023-05-07" xml:lang="de">enthält Daten zu einem Ereignis, das mit einer oder mehreren Personen,  Organisationen, Orten, Objekten oder anderen Ereignissen in Beziehung stehen kann.</desc>
+  <desc versionDate="2007-05-04" xml:lang="es">cualquier fenómeno o evento no necesariamente vocal o comunicativo, como ruidos accidentales u otros fenómenos que inciden en la comunicación en acto.</desc>
```
Comment (HelenaSabel) [inline P5/Source/Specs/event.xml]:
```suggestion
  <desc versionDate="2023-09-04" xml:lang="es">contiene datos relacionados con cualquier tipo de evento significativo asociado con una persona, lugar u organización.</desc>
```

**Comment by ebeshero** [comment]
Council F2F at Paderborn: We need to alter the content model of `<event>` to preserve backwards compatibility--at the moment, the current content model does not allow for `idno` followed by a `head` followed by a `p`.

So the content model will change like so: 

```xml 

      <elementRef key="idno" minOccurs="0" maxOccurs="unbounded"/>
      <classRef key="model.headLike" minOccurs="0" maxOccurs="unbounded"/>
      <alternate>
        <classRef key="model.pLike" minOccurs="1" maxOccurs="unbounded"/>
        <classRef key="model.labelLike" minOccurs="1" maxOccurs="unbounded"/>
        <elementRef key="eventName" minOccurs="1" maxOccurs="unbounded"/>
      </alternate>
      <alternate minOccurs="0" maxOccurs="unbounded">
        <classRef key="model.noteLike"/>
        <classRef key="model.biblLike"/>
        <elementRef key="linkGrp"/>
        <elementRef key="link"/>
        <elementRef key="idno"/>
        <elementRef key="ptr"/>
      </alternate>
<!-- [... listPerson etc ] -->
```

```
      idno*,
      model.headLike*,
      ( eventName+ | model.pLike+ | model.labelLike+ ),
      (
         model.noteLike
       | model.biblLike
       | linkGrp
       | link
       | idno
       | ptr
      )*,
 # listPerson etc
```

File: P5/Source/Specs/event.xml [hunk 4]
```diff
@@ -8,18 +8,17 @@
...
+  <desc versionDate="2023-05-07" xml:lang="fr">contient des données liées à tout type d'évènement significatif dans l'existence d'une personne, d'un lieu, d'un objet ou d'une organisation.</desc>
+  <desc versionDate="2007-01-21" xml:lang="it">qualsiasi fenomeno o evento non necessariamente vocale o comunicativo, come rumori accidentali o altri fenomeni che incidano sulla comunicazione in atto</desc>
```
Comment (raffazizzi) [inline P5/Source/Specs/event.xml]:
```suggestion
  <desc versionDate="2023-09-04" xml:lang="it">contiene dati a proposito di un qualsiasi tipo di evento significativo associato con una persona, luogo o organizzazione.</desc>
```

File: P5/Source/Guidelines/en/ND-NamesDates.xml [hunk 5]
```diff
@@ -2269,6 +2085,265 @@ occupation and residence.</p>
...
     </div>
 
+    <div xml:id="NDEVNT" type="div2">
+      <head>Events</head>
+      <p>
+        <specList>
+          <specDesc key="event"/>
+          <specDesc key="eventName"/>
+          <specDesc key="listEvent"/>
+        </specList> An event is a change of state whether real, planned, fictional, or purported about which more
```
Comment (ebeshero) [inline P5/Source/Guidelines/en/ND-NamesDates.xml:2095]: Council F2F in Paderborn: We can simplify this as simply: Broadly speaking, an event is something that happens, whether real, planned, fictional, or purported. @sydb @joeytakeda

File: P5/Source/Specs/event.xml [hunk 6]
```diff
@@ -8,18 +8,17 @@
 -->
 <?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
 <elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="event">
-  <gloss xml:lang="en" versionDate="2008-12-09">event</gloss>
+  <gloss versionDate="2008-12-09" xml:lang="en">event</gloss>
+  <gloss versionDate="2023-05-07" xml:lang="de">Ereignis</gloss>
   <gloss versionDate="2008-12-09" xml:lang="fr">évènement</gloss>
-  <desc versionDate="2007-08-01" xml:lang="en">contains data relating to any kind of significant event associated with a person, place, or organization.</desc>
-  <desc versionDate="2007-12-20" xml:lang="ko">개인, 장소, 또는 조직과 관련된 중요한 사건에 해당하는 데이터를 포함한다.</desc>
+  <desc versionDate="2023-05-07" xml:lang="en">contains data relating to any kind of significant event associated with a person, organization, place, object, or other event.</desc>
```
Comment (ebeshero) [inline P5/Source/Specs/event.xml:14]: Council F2F in Paderborn: Revise to simplify:
"contains data relating to anything of significance that happens in time."

File: P5/Source/Specs/eventName.xml [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="eventName">
+  <gloss versionDate="2023-05-02" xml:lang="en">name of an event</gloss>
+  <gloss versionDate="2023-05-02" xml:lang="de">Name eines Ereignisses</gloss>
+  <desc versionDate="2023-05-02" xml:lang="en">contains a proper noun or noun phrase used to refer to an event.</desc>
+  <desc versionDate="2023-05-02" xml:lang="de">enthält einen Eigennamen in Form eines Nomens oder einer Nominalphrase, der verwendet wird, um auf ein Ereignis zu verweisen.</desc>
+  <classes>
+    <memberOf key="att.datable"/>
+    <memberOf key="att.editLike"/>
+    <memberOf key="att.global"/>
```
Comment (joeytakeda) [inline P5/Source/Specs/eventName.xml:17]: Very small little copy editing thing: as discussed in F2F Paderborn, we tend to order `class/memberOf` with att.global first, and then the rest in alphabetical order.

File: P5/Source/Specs/eventName.xml [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+              <forename>Georg</forename>
+            </persName>
+          </person>
+        </listPerson>
+        <place xml:lang="de">
+          <placeName>Universität Graz</placeName>
+          <location>
+            <address>
+              <addrLine>ReSoWi Gebäude</addrLine>
+              <addrLine>Universitätsstratheße 15</addrLine>
```
Comment (martinascholger) [inline P5/Source/Specs/eventName.xml]:
```suggestion
              <addrLine>Universitätsstraße 15</addrLine>
```

**Comment by joeytakeda** [comment]
Thanks very much indeed for all your work on this issue, @skurzinz🙂 — we are currently merging into a feature branch to address a few small changes before we merge fully into the `dev` branch.

**Comment by skurzinz** [comment]
Just for reference I am adding the follow-up issue #2499

**Pull request merged**